.travis.yml

```diff
@@ -86,13 +86,15 @@ install:
         && cd petsc-3.10.0
         && ./configure --with-debugging=0 COPTFLAGS=-O --with-mpi=0 --with-fc=0 --with-cxx=0 --prefix=$PETSC_INSTALL
         && make
-        && make install )
+        && make install
+        && cd .. )
```

Comment:
The `( list )` syntax runs in a sub-shell. The `cd ..` shouldn't be needed. Why was this added?

Can we use a sub-shell below as well?

Reply:
Just a mistake from my ignorance of the sub shell syntax , fixed up now.